src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+import java.io.File;
```

Comment:
Perhaps you can have a line of space between the package and the import? Personally, I think it would have better readability. The same goes for all the other files.